x-pack/legacy/plugins/apm/public/utils/formatters.ts

```diff
@@ -245,3 +246,47 @@ export const getFixedByteFormatter = memoize((max: number) => {
 
   return bailIfNumberInvalid(formatter);
 });
+
+function twoWithSameValue(ticks: Moment[], format: string) {
+  return ticks
+    .map(t => t.format(format))
+    .reduce((acc, val) => ({ same: acc.same || acc.prev === val, prev: val }), {
```

Comment:
This fails type checking because `reduce` infers the type of the second argument `{ same: boolean }` and we're referencing the `prev` property here.

One way to fix this is by giving a type parameter to `reduce`:

```suggestion
    .reduce<{ prev?: string; same: boolean }>(
      (acc, val) => ({ same: acc.same || acc.prev === val, prev: val }),
      {
```